turbo.json

```diff
@@ -20,6 +20,7 @@
       "dependsOn": ["build"],
       "outputs": ["coverage/**"],
       "inputs": ["src/**/*.tsx", "src/**/*.ts"]
-    }
+    },
+    "lint": {}
```

Comment:
Does `lint` depend on `build` now?

Reply:
I thought it should, but since we exclude `dist` everywhere, it shouldn't need to depend on `build`.

Reply:
FWIW, the official [eslint example](https://turbo.build/repo/docs/guides/tools/eslint) also doesn't have it depending on anything

Reply:
Hm.  We exclude `dist` from list of files to be linted, but some ESLint rules depend on `.d.ts` files.  

In particular, the `no-floating-promises` rule might depend on build output.